code_samples/api/commerce/src/Command/ShipmentCommand.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+ 
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Ibexa\Contracts\Shipping\ShipmentServiceInterface;
```

Comment:
```suggestion
use Ibexa\Contracts\Checkout\ShipmentServiceInterface;
```
Currently, namespaces are not `Shipping` but `Checkout`